app/src/main/java/com/better/alarm/presenter/AlarmsListActivity.kt

```diff
@@ -189,13 +203,52 @@ class AlarmsListActivity : AppCompatActivity() {
     override fun onStart() {
         super.onStart()
         configureTransactions()
+        configureSnackbar()
+    }
+
+    private fun configureSnackbar() {
+        val rootView = window.decorView.rootView
+
+
+
+
+
+        store.sets().withLatestFrom<Boolean, Pair<Store.AlarmSet, Boolean>>(
+            store.uiVisible,
+            BiFunction { set, uiVisible -> set to uiVisible })
+            .subscribe { (set: Store.AlarmSet, uiVisible: Boolean) ->
```

Comment:
Leaked disposable here. This subscription will not be disposed when the Fragment is destroyed

- [ ] Add disposable to the composite disposable of the fragment

Reply:
This code is in the activity

Reply:
Done . i added it in the onStop of the activity